src/Drivers/Sqlite.php

```
@@ -0,0 +1,123 @@
+
+            $this->conn->query('CREATE TABLE `' . $table . '` (
+                `id` TEXT PRIMARY KEY,
+                `data` TEXT NOT NULL,
+                `time` INTEGER NOT NULL
+            )');
+        }
+    }
+
+    public function open($path, $name): bool
+    {
+        return true;
+    }
+
+    public function close(): bool
+    {
+        return true;
+    }
+
+    public function read($id): string
+    {
+        $data = '';
+        $statement = $this->conn->prepare("SELECT `data` FROM `{$this->table}` WHERE `id` = :id");
+        $statement->bindParam(':id', $id, PDO::PARAM_STR);
+
+        if ($statement->execute()) {
+            $result = $statement->fetch();
+            $data = $result['data'] ?? '';
+        } else {
+            // Debug information
+            error_log("Failed to execute read statement for ID: {$id}");
+        }
+
+        $statement = null; // close
+        return $this->get($data);
+    }
+
+    public function write($id, $data): bool
+    {
+        $statement = $this->conn->prepare("REPLACE INTO `{$this->table}` (`id`, `data`, `time`) VALUES (:id, :data, :time)");
+        $statement->bindParam(':id', $id, PDO::PARAM_STR);
+        $statement->bindValue(':data', $this->set($data), PDO::PARAM_STR);
+        $statement->bindValue(':time', time(), PDO::PARAM_INT);
+
+        $completed = $statement->execute();
+        if (!$completed) {
+            // Debug information
+            error_log("Failed to execute write statement for ID: {$id}");
+        }
+
+        $statement = null; // close
+        return $completed;
+    }
+
+    public function destroy($id): bool
+    {
+        $statement = $this->conn->prepare("DELETE FROM `{$this->table}` WHERE `id` = :id");
+        $statement->bindParam(':id', $id, PDO::PARAM_STR);
+
+        $completed = $statement->execute();
+        if (!$completed) {
+            // Debug information
+            error_log("Failed to execute destroy statement for ID: {$id}");
```

Lets throw an exception instead of error_log